pages/Login/signIn.tsx

```diff
@@ -0,0 +1,15 @@
+import style from "./Login.module.scss";
+import SocialLogin from "../../src/components/pageComponent/login/socialLogin";
+import LoginBox from "../../src/components/pageComponent/login/loginBox";
+
+function SignIn() {
+  return (
+    <div className={style.login}>
+      <div className={style.loginFrame}>
+        <LoginBox />
```

Comment:
이렇게 표현해주는게 의미가 더 좋을 것 같아요~
```suggestion
        <LoginForm />
```

Reply:
바꿨습니다